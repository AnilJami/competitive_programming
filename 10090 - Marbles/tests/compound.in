a
alien
alight
alive
born
cat
catnap
less
lien
light
live
nap
never
nevertheless
new
newborn
so
soso
sososo
the
tube
tubelight
u
utube
wrap
wrapper
zebra
