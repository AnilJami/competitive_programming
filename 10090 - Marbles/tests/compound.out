alien
alight
alive
catnap
newborn
soso
sososo
tubelight
utube
